.github/workflows/check-release-branch.yml

```diff
@@ -38,3 +38,55 @@ jobs:
                     echo "Base branch has commits since merge base"
                     exit 1
                   fi
+
+    check_pr_branch:
+        name: Check PR branch
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Get branch name of the PR
+              id: pr_branch_name
+              run: |
+                  pr_branch_name=$(git branch --show-current)
+                  echo "result=$PR_BRANCH_NAME" >> $GITHUB_OUTPUT
```

Comment:
there might be a better way of doing this

Reply:
does parsing the version (used in other steps) need checkout though?

Reply:
yes